.buildkite/pipeline.yml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: 'load-test'
```

Comment:
Wouldn't you rather have separate build and test steps?

Reply:
I kinda would, but the way this docker-compose plugin works it's just much easier to do it this way

Reply:
It's faster now but once we add more parallel steps the build agents will slow down.